flamedisx/source.py

```diff
@@ -205,8 +217,17 @@ def __init__(self,
         """
         if max_sigma is None:
             max_sigma = self.default_max_sigma
+        self.bounds_prob = 0.5 * (1 + sp.erf(-max_sigma / np.sqrt(2)))
```

Comment:
`stats.norm.cdf(-self.max_sigma)` would be clearer (with `from scipy import stats` in the imports of course)